### PR TITLE
FIX(cherry-pick): Invalid configs can lead to validation throwing a 500 error.

### PR DESCRIPTION
**Pull request by snehashisp**
Cherry pick https://github.com/confluentinc/kafka-connect-elasticsearch/pull/626 to 11.1.x.

##### Does this solution apply anywhere else?
- [ ] yes
- [x] no

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [x] Manual tests

### CODE REVIEW

**Comment by pbadani** [comment]
Any of the validations moved out of try catch don't throw an exception?

File: src/main/java/io/confluent/connect/elasticsearch/Validator.java [hunk 1]
```diff
@@ -100,25 +100,27 @@ public Config validate() {
...
-      if (!hasErrors()) {
-        // no point if previous configs are invalid
+    validateCredentials();
+    validateDataStreamConfigs();
+    validateIgnoreConfigs();
+    validateKerberos();
+    validateLingerMs();
+    validateMaxBufferedRecords();
+    validateProxy();
+    validateSsl();
```
Comment (pbadani) [inline src/main/java/io/confluent/connect/elasticsearch/Validator.java:110]: Any of the validations moved out of try catch don't throw an exception?

Comment (snehashisp) [reply src/main/java/io/confluent/connect/elasticsearch/Validator.java:110]: None from the looks of things. Also earlier, only IO exception related to closing client was handled so I presume these ones don't have any scope to throw any exception.

Comment (pbadani) [reply src/main/java/io/confluent/connect/elasticsearch/Validator.java:110]: ok

**Review by pbadani — APPROVED** [review]

**Pull request merged**